store.js

```diff
@@ -48,19 +48,30 @@ assign(Store.prototype, {
 	_sortComputedProperties: function() {
 		var computed = this._computed;
 		var sorted = this._sortedComputedProperties = [];
+		var cycles;
 		var visited = blankObject();
 
 		function visit(key) {
+			if (cycles[key]) {
+				throw new Error(`Cyclical dependency detected — a computed property cannot indirectly depend on itself`);
```

Comment:
Probably shouldn't use template literals in code that's going to get shipped to users.

It's also bugging me a bit having this whole error message text end up in production code, but I don't see a reasonable way around that.

Reply:
Oops, good catch. Should perhaps add a test for ES6+isms. I agree about the error message. Maybe we could just shorten it? e.g. just 'Cyclical dependency detected'

Reply:
:+1: Using a shorter error message seems like a reasonable compromise.